<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Outils</class>
 <widget class="QDialog" name="Outils">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>751</width>
     <height>361</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>110</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(107, 35, 131);</string>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(107, 35, 131);</string>
     </property>
     <property name="text">
      <string>MOT DE PASSE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>190</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(107, 35, 131);</string>
     </property>
     <property name="text">
      <string>CONFIRMER MOT DE PASSE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(107, 35, 131);</string>
     </property>
     <property name="text">
      <string>MAIL DE RECUPERATION</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mdp">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 255);
color: rgb(0, 0, 0);
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mdp2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>190</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 255);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_valider_inscription">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>300</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>VALIDER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>471</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 255);
font: 75 20pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>INSCRIPTION/CREATION DE LOGIN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-190</y>
       <width>751</width>
       <height>631</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../resources/Pink and Purple Beauty and Fashion Instagram Post(2).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_verification_infos">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>280</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(170, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_mail_inscris">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>230</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <zorder>label_12</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>lineEdit_username</zorder>
    <zorder>lineEdit_mdp</zorder>
    <zorder>lineEdit_mdp2</zorder>
    <zorder>pushButton_valider_inscription</zorder>
    <zorder>label_5</zorder>
    <zorder>label_verification_infos</zorder>
    <zorder>comboBox_mail_inscris</zorder>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-190</y>
       <width>751</width>
       <height>631</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../resources/Pink and Purple Beauty and Fashion Instagram Post(2).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>551</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 255);</string>
     </property>
     <property name="text">
      <string>REINITIALISATION DE MOT DE PASSE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>220</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
color: rgb(107, 35, 131);</string>
     </property>
     <property name="text">
      <string>ENTREZ LE MAIL DE RECUPERATION</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
color: rgb(107, 35, 131);</string>
     </property>
     <property name="text">
      <string>ENTREZ LE USERNAME/LOGIN</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>140</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>220</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>310</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Reinitialisez le mot de passe</string>
     </property>
     <property name="text">
      <string>M'ENVOYER UN MOT DE PASSE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-190</y>
       <width>751</width>
       <height>631</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../resources/Pink and Purple Beauty and Fashion Instagram Post(2).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_11</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>lineEdit_5</zorder>
    <zorder>lineEdit_6</zorder>
    <zorder>pushButton_2</zorder>
   </widget>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>AIDES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reinitialiser_mdp">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>RECUPERER MOT DE PASSE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_inscription">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AJOUTER UN COMPTE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
